File: python/functions/string_manipulation.py
#!/usr/bin/env python3

text = "  Hello, World!  "

print(f"Original: '{text}'")
print(f"Upper: '{text.upper()}'")
print(f"Lower: '{text.lower()}'")
print(f"Strip: '{text.strip()}'")
print(f"Replace: '{text.replace('World', 'Python')}'")
print(f"Split: {text.strip().split(', ')}")
print(f"StartsWith: {text.strip().startswith('Hello')}")
print(f"EndsWith: {text.strip().endswith('!')}")

File: ruby/functions/string_manipulation.rb
#!/usr/bin/env ruby

text = "  Hello, World!  "

puts "Original: '#{text}'"
puts "Upper: '#{text.upcase}'"
puts "Lower: '#{text.downcase}'"
puts "Strip: '#{text.strip}'"
puts "Replace: '#{text.gsub('World', 'Ruby')}'"
puts "Split: #{text.strip.split(', ')}"
puts "Include?: #{text.include?('World')}"
puts "Start with?: #{text.strip.start_with?('Hello')}"
puts "End with?: #{text.strip.end_with?('!')}"
puts "Length: #{text.strip.length}"
puts "Repeat: '##{'-' * 20}'"
puts "Reverse: '#{text.strip.reverse}'"
